php/composer.json

```
@@ -0,0 +1,6 @@
+{
+  "require": {
+    "guzzlehttp/guzzle": "^7.0",
+    "vlucas/phpdotenv": "^5.5"
```

```suggestion
    "guzzlehttp/guzzle": "^7.0",
    "guzzlehttp/psr7": "^2.4",
    "vlucas/phpdotenv": "^5.5"
```
(Adds utility classes for formatting requests/responses)